### DISCUSSION TITLE
Z2M sonoff OTA

### DISCUSSION THREAD

**Discussion by marinus61** [Koenkk/zigbee2mqtt#13762] [Devices]
Hi out there

in get messages that there are OTA available for my Sonoff units. HOwever, the status stays that way for days.

Read the OTA update page and many others, and found the command:

zigbee2mqtt/bridge/request/device/ota_update/check id: friendly_name

but this is not a shell command, so from where do you run it. Is it to be loaded into the configuration.yaml of Z2M or.

Thanks for helping out
Marinus

**Comment by Koenkk** [comment]
It is an MQTT command, alternately you can open the z2m frontend -> OTA and update from there.

**Reply by marinus61** [reply]
@Koenkk

1) how do I send a MQTT command
2) how do I open the frontend of Z2M? I have tried to set frontend: true in Configuration.yaml and during startup log says frontend started 0.0.0.0:8099

Z2M is new to me for HA, so I am little rusty here.

Marinus

**Reply by Koenkk** [reply]
1. You can use e.g. http://mqtt-explorer.com/
2. Use your browser and go to `COMPUTER_IP:8099`, if you are running via the HA addon, go tot he addon config -> show in side bar and click the sidebar icon in the left HA menu.

**Reply by marinus61** [reply]
@Koenkk 

thks for the info. Installed mqtt explorer, and it is running

I tried the command as described in Z2M OTA update under device, but no reaction
{ zigbee2mqtt/bridge/request/device/ota_update/check {id: "Terrasse gæst gulv" }} 

Could you be so kind as to assist me what I am doing wrong with the command?

Marinus
